packages/block-editor/src/experiments.js

```diff
@@ -22,4 +23,5 @@ export const experiments = {};
 lock( experiments, {
 	...globalStyles,
 	ExperimentalBlockEditorProvider,
+	__experimentalOffCanvasEditor,
```

Comment:
now that this is actually "locked", there's no need for `__experimental` prefix.

Reply:
Should I go for the naming in `ExperimentalBlockEditorProvider` and use `ExperimentalOffCanvasEditor`?

Reply:
I think `ExperimentalBlockEditorProvider` only has a prefix because there's also the stable one `BlockEditorProvider`. So for the off canvas, I think it should be just `OffCanvasEditor`

Reply:
Riad is right! You can just rename to `OffCanvasEditor`.

Reply:
Done in #47553